python/Dockerfile.fedora.python3.slim.tpl

```diff
@@ -0,0 +1,80 @@
+FROM #{FROM}
+
+# http://bugs.python.org/issue19846
+# > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
+ENV LANG C.UTF-8
+
+# install python dependencies
+RUN rpm --rebuilddb && yum install -y \
+		libsqlite3x \
+		openssl-devel \
+	&& yum clean all
+
+# key 63C7CC90: public key "Simon McVittie <smcv@pseudorandom.co.uk>" imported
+RUN gpg --keyserver keyring.debian.org --recv-keys 4DE8FF2A63C7CC90
+
+# key 3372DCFA: public key "Donald Stufft (dstufft) <donald@stufft.io>" imported
+RUN gpg --keyserver pgp.mit.edu  --recv-key 6E3CBCE93372DCFA
+
+ENV PYTHON_VERSION #{PYTHON_VERSION}
+
+# if this is called "PIP_VERSION", pip explodes with "ValueError: invalid truth value '<VERSION>'"
+ENV PYTHON_PIP_VERSION 8.1.2
+ENV PYTHON_PIP_SHA256 8dae1fb72e29c2b6ff6ed267861179216bf98d3bda6d30e527dbed0db5ac7e1d
+
+ENV SETUPTOOLS_SHA256 3f24380f0b18f869b7ad92d9e16d1d58093a9bac13bbd0b0e1fc5a82b460311f
+ENV SETUPTOOLS_VERSION 21.0.0
+
+RUN set -x \
+	&& buildDeps=' \
+		curl \
+	' \
+	&& rpm --rebuilddb && yum install -y $buildDeps && yum clean all \
+	&& curl -SLO ftp://195.220.108.108/linux/opensuse/ports/update/13.1/armv7hl/libopenssl1_0_0-1.0.1e-11.9.1.armv7hl.rpm \
+	&& echo "73ad776d7a6cc5d2559b209499ae4b82d0ce329d2f9e2ce345d249350c34bf45  libopenssl1_0_0-1.0.1e-11.9.1.armv7hl.rpm" | sha256sum -c - \
+	&& rpm -ivh libopenssl1_0_0-1.0.1e-11.9.1.armv7hl.rpm \
+	&& rm libopenssl1_0_0-1.0.1e-11.9.1.armv7hl.rpm \
+	&& curl -SLO "#{BINARY_URL}" \
+	&& echo "#{CHECKSUM}" | sha256sum -c - \
+	&& tar -xzf "Python-$PYTHON_VERSION.linux-#{TARGET_ARCH}.tar.gz" --strip-components=1 \
+	&& rm -rf "Python-$PYTHON_VERSION.linux-#{TARGET_ARCH}.tar.gz" \
+	&& echo "/usr/local/lib" >> /etc/ld.so.conf \
+	&& ldconfig \
+	&& mkdir -p /usr/src/python/setuptools \
+	&& curl -SLO https://github.com/pypa/setuptools/archive/v$SETUPTOOLS_VERSION.tar.gz \
+	&& echo "$SETUPTOOLS_SHA256  v$SETUPTOOLS_VERSION.tar.gz" > setuptools-$SETUPTOOLS_VERSION.tar.gz.sha256sum \
+	&& sha256sum -c setuptools-$SETUPTOOLS_VERSION.tar.gz.sha256sum \
+	&& tar -xzC /usr/src/python/setuptools --strip-components=1 -f v$SETUPTOOLS_VERSION.tar.gz \
+	&& rm -rf v$SETUPTOOLS_VERSION.tar.gz* \
+	&& cd /usr/src/python/setuptools \
+	&& python3 setup.py install \
+	&& mkdir -p /usr/src/python/pip \
+	&& curl -SL "https://github.com/pypa/pip/archive/$PYTHON_PIP_VERSION.tar.gz" -o pip.tar.gz \
+	&& echo "$PYTHON_PIP_SHA256  pip.tar.gz" > pip.tar.gz.sha256sum \
+	&& sha256sum -c pip.tar.gz.sha256sum \
+	&& tar -xzC /usr/src/python/pip --strip-components=1 -f pip.tar.gz \
+	&& rm pip.tar.gz* \
+	&& cd /usr/src/python/pip \
+	&& python3 setup.py install \
+	&& cd .. \
+	&& find /usr/local \
+		\( -type d -a -name test -o -name tests \) \
+		-o \( -type f -a -name '*.pyc' -o -name '*.pyo' \) \
```

Comment:
those are compiled python scripts and can be removed after finishing building setuptools and pip